packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -478,7 +478,15 @@ export default {
     throw new Error('non pool-wide install not implemented')
   },
 
-  async rollingPoolUpdate() {
+  @decorateWith(deferrable)
+  async rollingPoolUpdate($defer) {
+    if (this.pool.ha_enabled) {
+      const haSrs = this.pool.$ha_statefiles.map(vdi => vdi.SR)
```

Comment:
It's weird that XAPI doc shows `ha_statefiles` as a `string set` instead of a `VDI ref set`…

Do you think this might bites us in the ass?

Reply:
The description does say "HA statefile VDIs in use" so I guess it means that the strings can only be VDI refs.

Reply:
Fine by me :shrug: :sweat_smile: